test/cluster/query-without-default-cluster.td

```diff
@@ -0,0 +1,30 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ postgres-connect name=mz_system url=postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}
+
+$ postgres-execute connection=mz_system
+DROP CLUSTER default CASCADE
+
+> SELECT 0
+0
+
+> SELECT 1 as id
+1
+
+> SELECT 1 from (select mz_now(), now())
+1
+
+# filtering isn't supported as a simple query today
+! SELECT * FROM (VALUES (1), (2)) where column1 > 1
+contains:unknown cluster 'default'
+
+# to make it easier to run the test again, let's recreate the default cluster
+$ postgres-execute connection=mz_system
+CREATE CLUSTER default REPLICAS ()
```

Comment:
I think this is already done automatically as part of the test framework. @def- does that sound correct?

Reply:
I added this when I was running `env COCKROACH_URL=postgres://root@localhost:26257 cargo run --bin testdrive -- test/cluster/query-without-default-cluster.td --default-timeout=1s` back to back. testdrive alone won't create the default cluster (since it usually exists)